<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DenasGUI</class>
 <widget class="QMainWindow" name="DenasGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DenasGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">powerButton:clicked{
color:#fff;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed { background-color: #808080; }</string>
   </property>
   <widget class="QProgressBar" name="batteryBar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skinSimulator">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Skinpoint</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;center&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;POWER LOW &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="timerscreen">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView&gt;QLCDNumber{
	color: red
	font-size: 15px;
	width: 30px;
 	height: 30px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="colon">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>64</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>batteryBar</zorder>
   <zorder>upButton</zorder>
   <zorder>downButton</zorder>
   <zorder>leftButton</zorder>
   <zorder>rightButton</zorder>
   <zorder>listWidget</zorder>
   <zorder>powerButton</zorder>
   <zorder>backButton</zorder>
   <zorder>menuButton</zorder>
   <zorder>okButton</zorder>
   <zorder>skinSimulator</zorder>
   <zorder>warning</zorder>
   <zorder>timerscreen</zorder>
   <zorder>colon</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
